codegens/http/lib/util.js

```diff
@@ -117,6 +117,39 @@ function convertPropertyListToString (propertyList, joinUsing, includeDisabled =
   }), joinUsing);
 }
 
+/**
+ * parses variable of request url object and sets hostname, path and query in request object
+ *
+ * @param {Object} request - Postman SDK request object
+ */
+function parseURLVariable (request) {
+  const variableArray = _.get(request.toJSON(), 'url.variable', []);
+
+  if (!variableArray.length) {
+    return;
+  }
+
+  variableArray.forEach(function (variableArrayElement) {
+    request.url.host.forEach(function (hostArrayElement, hostArrayElementIndex) {
+      if (hostArrayElement === ':' + variableArrayElement.key) {
+        request.url.host[hostArrayElementIndex] = variableArrayElement.value;
+      }
+    });
+
+    request.url.path.forEach(function (pathArrayElement, pathArrayElementIndex) {
```

Comment:
we only need to check for url.path, right? :path variables don't exist in the host or query params

Reply:
ah, yes. done